pkg/utils/utils.go

```diff
@@ -212,3 +212,22 @@ func DeDuplicate(input []string) []string {
 	}
 	return result
 }
+
+// FormatErrorFunc is a function which used by CombineErrors to
+// format error message
+type FormatErrorFunc func(idx int, err error) (string, error)
+
+// CombineErrors is a function which used by MultiInspect to merge multiple errors
+// into one error.
+func CombineErrors(errs []error, formatError FormatErrorFunc) error {
+	var errMsgs []string
+	for idx, err := range errs {
+		formattedErrMsg, formatError := formatError(idx, err)
```

Comment:
I think we could improve this part, since this could be ambiguous that two `formatError ` representing different type exist there.

Could we just change a naming?

Reply:
Thanks for your reminder.
I have changed it.

Reply:
There are only two hard things in Computer Science: cache invalidation and naming things.

-- Phil Karlton

😄 🌹

Reply:
:+1:
And are there any suggestions for better naming? I can't find a better one.
Thanks a lot for your comments.